File: ABC/090/b.cs
using System;
using System.Linq;

class Program
{
    static void Main(string[] args) {
        int[] s = Console.ReadLine().Split().Select(int.Parse).ToArray();
        Console.WriteLine(solve(s[0], s[1]));
    }
    static int solve(int a, int b) {
        int count = 0;
        for(int i = a; i <= b; i ++) {
            char[] c = i.ToString().ToCharArray();
            Array.Reverse(c);
            string s = new string(c);
            int tmp = int.Parse(s);
            if(i == tmp) count ++;
        }
        return count;
    }
}

File: ABC/090/b.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int a,b; cin >> a >> b;
    int count = 0;
    for(int i = a; i <= b; i++) {
        string s = to_string(i);
        if(s[0]==s[4] && s[1]==s[3]) count ++;
    }
    cout << count << endl;
}